.github/workflows/blogs-indexing.yaml

```diff
@@ -0,0 +1,46 @@
+name: Indexing blogs
+on:
+  push:
+    branches: ['search-blogs-ci-indexing']
+    paths:
+      - 'content/blog/**'
+jobs:
+  build-and-indexing-blogs:
+    name: Build and indexing blogs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Node.js and use yarn
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14.x'
+      - name: create env file
+        run: |
+          touch .env
+          echo GATSBY_ALGOLIA_APP_ID=${{ secrets.GATSBY_ALGOLIA_APP_ID }} >> .env
+          echo ALGOLIA_ADMIN_KEY=${{ secrets.ALGOLIA_ADMIN_KEY }} >> .env
+          echo GATSBY_ALGOLIA_INDEX_NAME=${{ secrets.GATSBY_ALGOLIA_INDEX_NAME }} >> .env
+      - name: Yarn cache directory
+        id: yarn-cache-dir
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - name: Gatsby Cache Folder
+        uses: actions/cache@v2
+        with:
+          key: gatsby-cache-folder
+          path: .cache
+      - name: Gatsby Public Folder
+        uses: actions/cache@v2
+        with:
+          key: gatsby-public-folder
+          path: public
+      - name: Yarn cache
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.yarn-cache-dir.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
```

Comment:
Caching yarn to GitHub actions for faster builds.

Reply:
There's [a built-in caching feature for `setup-node`](https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#caching-packages-dependencies) that takes care of proper implementation for this, though granted we haven't used it up until now.